base/docs/helpdb/Base.jl

```diff
@@ -1767,16 +1767,19 @@ deserialize
 """
     length(collection) -> Integer
 
-For ordered, indexable collections, returns the maximum index `i` for which `getindex(collection, i)`
-is valid.
-For unordered collections, returns the number of elements.
+Return the number of elements in collection.
```

Comment:
I'm confused. If its not referencing the argument, should it be `a collection` or `the collection` then (its `a collection` in the sentence below). Is it not better to disambiguate like this:

```julia
"""
    length(A) -> Integer

Return the number of elements in [collection] `A`.

Use [`endof`](@ref) to get the last valid index of an indexable collection.
"""